modules/data-model/forms/api/src/main/java/io/uhndata/cards/forms/api/FormUtils.java

```diff
@@ -434,4 +436,13 @@
      *         value is stored in the answer
      */
     Object getValue(NodeState answer);
+
+    /**
+     * Serialize the value(s) stored in an Answer.
+     *
+     * @param property a property of Answer node, may be {@code null}
+     * @return the {@code JsonValue} or values stored in the answer; {@code null} may be returned if no
+     *         value is stored in the answer or in case of exception caught in serialization process
```

Comment:
> {@code null} may be returned

Not exactly true, this returns `JsonValue.NULL` which is a special marker for null values, but not null itself.


```suggestion
     * @return a {@code JsonValue} holding the value or values stored in the answer; this may be a single value or an
           array of values, or {@code JsonValue.NULL} if no value is stored in the answer or in case of exception
           caught in the serialization process
```